### PR TITLE
Make DateTimePicker actually pick times too.

### PR DESCRIPTION
**Pull request by ashb**
This is reported upstreadm as dpgaspar/Flask-AppBuilder#685 but there
appears to be 0 appetite for fixing it there. Since we can't easily
change the code in FAB I have had to customize the Widget, and give it a
different class name

https://github.com/eonasdan/bootstrap-datetimepicker/ is MIT licensed.

**Before:**

image

**After:**

image

---
Issue link: WILL BE INSERTED BY [boring-cyborg](https://github.com/kaxil/boring-cyborg)

Make sure to mark the boxes below before creating PR: [x]

- [x] Description above provides context of the change
- [x] Unit tests coverage for changes (not needed for documentation changes)
- [x] Commits follow "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)"
- [x] Relevant documentation is updated including usage instructions.
- [x] I will engage committers as explained in [Contribution Workflow Example](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#contribution-workflow-example).

---
In case of fundamental code change, Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in [UPDATING.md](https://github.com/apache/airflow/blob/master/UPDATING.md).
Read the [Pull Request Guidelines](https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#pull-request-guidelines) for more information.

### CODE REVIEW

File: airflow/www/forms.py [hunk 1]
```diff
@@ -33,18 +33,20 @@
 from airflow.utils import timezone
 from airflow.www.validators import ValidJson
 
+from .widgets import AirflowDateTimePickerWidget
```
Comment (kaxil) [inline airflow/www/forms.py]:
```suggestion
from airflow.www.widgets import AirflowDateTimePickerWidget
```

Comment (ashb) [reply airflow/www/forms.py]: I guess. I do wonder why we don't use relative imports more often...

Comment (kaxil) [reply airflow/www/forms.py]: I am not sure either. But based on [PEP 8](https://www.python.org/dev/peps/pep-0008/):

>Absolute imports are recommended, as they are usually more readable and tend to be better behaved (or at least give better error messages) if the import system is incorrectly configured (such as when a directory inside a package ends up on sys.path):

>However, explicit relative imports are an acceptable alternative to absolute imports, especially when dealing with complex package layouts where using absolute imports would be unnecessarily verbose:

So 🤷‍♂

Comment (ashb) [reply airflow/www/forms.py]: Yeah, the "explicit relative imports" bit is this. In this case it's not overly verbose without it so I guess for consistency i should use absolute.

Comment (kaxil) [reply airflow/www/forms.py]: Yup

**Review by kaxil — APPROVED** [review]

**Pull request merged**